gnocchi/rest/api.py

```diff
@@ -1542,16 +1546,32 @@ def post(self, metric_id, start=None, stop=None, aggregation='mean',
 
 
 class ResourcesMetricsMeasuresBatchController(rest.RestController):
+
+    @staticmethod
+    def BackwardCompatibleMeasuresList(v):
+        v = voluptuous.Schema(
+            voluptuous.Any(MeasuresListSchema,
+                           {voluptuous.Optional("archive_policy_name"):
+                            six.text_type,
+                            voluptuous.Optional("unit"):
```

Comment:
is unit here because in case it needs to be created?

i'll be honest, i don't really like the unit attribute... to me it's equivalent to having a description attribute... but oh well.

Reply:
I don't really get the use case behind unit too. I just need to be able to fill any metric attributes here for Ceilometer and unit is just one of them.